imap_processing/codice/constants.py

```diff
@@ -210,6 +209,158 @@
     if is_active
 ]
 
+# Energy tables for CoDICE-Hi data products. These values represent the edges
```

Comment:
Just moving this up higher in the file so that it can be referenced in that data product configuration dictionary